setup.py

```diff
@@ -36,7 +36,6 @@
     ),
     extras_require={
         "docs": ("sphinx", "pallets-sphinx-themes"),
-        "lint": ("pre-commit ~= 1.17"),
```

Comment:
this didn't seem to be used anywhere... and pre-commit installs everything it needs anyway, so i don't think we need anything more here